docs/releasing/publishing-from-a-support-branch.md

```diff
@@ -100,17 +99,24 @@ Note: Before you go on annual leave, tell the delivery manager who will be looki
 
   You will now be prompted to continue or cancel.
 
-9. Raise a pull request, with `support/<CURRENT MAJOR VERSION NUMBER>.x` as the base branch to merge into.
+9. Raise a pull request, with `support/<MAJOR VERSION NUMBER>.x` as the base branch to merge into.
 
-10. Once a developer approves the pull request, merge it into `support/<CURRENT MAJOR VERSION NUMBER>.x`.
+10. Once a developer approves the pull request, merge it into `support/<MAJOR VERSION NUMBER>.x`.
 
 ### Publish the release to npm
 
-1. Check out `support/<CURRENT MAJOR VERSION NUMBER>.x` and pull the latest changes.
+1. Check out `support/<MAJOR VERSION NUMBER>.x` and pull the latest changes.
 
 2. Sign in to npm (`npm login`), using the npm/govuk-patterns-and-tools team [credentials](https://github.com/alphagov/design-system-team-credentials/tree/main/npm/govuk-patterns-and-tools).
 
-3. Run `npm run publish-release`, which will prompt you to either continue or cancel. Enter `y` to continue.
+3. Run `npm run publish-release`, which will prompt you to check whether the NPM tag looks as expected.
+
+  If you're following these instructions, it's likely that you're publishing an out-of-order release which should
+  *not* be tagged as the 'latest' release on NPM. In this scenario, the tag should take the format `latest-[major-version-number`, for example: `latest-4`.
```

Comment:
```suggestion
  Do not tag this release as the 'latest' release on npm. Instead, give the tag the format `latest-[major-version-number`, for example,`latest-4`.
```

- Does there need to be a ']' after 'number'?
- What's an 'out-of-order' release? Will the users of this doc know?

Reply:
> Does there need to be a ']' after 'number'?

Thanks, good spot!

> What's an 'out-of-order' release? Will the users of this doc know?

Very good question. I struggled with how to describe this kind of release. I don't think it's clear from 'out-of-order' but wasn't sure how else to describe it. There's a thread on Slack here about it: https://gds.slack.com/archives/CAF8HTJES/p1653049737575209 Happy to hear any thoughts/suggestions!

I'm also going to change the wording because actually I don't think it is likely that the person will be doing one - it's just a lot more likely than if they were following the 'publishing.md' docs.